wiseService/package.json

```diff
@@ -17,7 +17,7 @@
     "build": "npm run bundle:min",
     "start": "node wiseService.js -c ../tests/config.test.ini",
     "prestart": "npm run bundle:min",
-    "dev": "nodemon wiseService.js -c ../tests/config.test.ini & webpack -d --watch --progress --hide-modules --display-error-details --config vueapp/build/webpack.dev.conf.js",
+    "dev": "nodemon wiseService.js -c ../tests/config.test.json & webpack -d --watch --progress --hide-modules --display-error-details --config vueapp/build/webpack.dev.conf.js",
```

Comment:
until editing is available lets use the ini one

Reply:
Ok